src/nix/hive.rs

```diff
@@ -94,7 +94,7 @@ impl Hive {
 
     /// Retrieve deployment info for a single node.
     pub async fn deployment_info_for(&self, node: &str) -> NixResult<Option<NodeConfig>> {
-        let expr = format!("toJSON (hive.nodes.\"{}\".config.deployment or null)", node);
+        let expr = format!("builtins.toJSON (hive.nodes.\"{}\".config.deployment or null)", node);
```

Comment:
In the non-flake case `toJSON` is pulled in with `with builtins;`, and I initially did it to make it more ergonomic to write short `introspect` expressions. Let's decide whether to remove it to keep things consistent.

Reply:
I think we should be able to add it back to the flake evaluator without issues, I agree with you that it's useful for writing shorter expressions.